### PR TITLE
Add Odoo to EasyPanel!

### PR DESCRIPTION
**Pull request by DrMxrcy**

### CODE REVIEW

File: templates/odoo/index.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        {
+          type: "volume",
+          name: "odoo-addons",
+          mountPath: "/mnt/extra-addons",
+        },
+      ],
+    },
+  });
+  services.push({
+    type: "app",
```
Comment (deiucanta) [inline templates/odoo/index.ts:50]: Any reason for not using the dedicated service for postgres?

**Comment by deiucanta** [comment]
The `yarn.lock` file changes keep coming back. In order to not have it in the PR, you need to not push the changes. In this PR you deleted the file and pushed that. It's still a change with the file. You can try to undo your commit and revert any changes to the lock file. If you have issues, I can do it on my local machine and merge it there.

File: templates/odoo/index.ts (hunk 1 above)
Comment (DrMxrcy) [reply templates/odoo/index.ts:50]: The reason is becuase Odoo would not work with the postgres username, and I saw no way to change it with the dedicated service

**Comment by deiucanta** [comment]
@DrMxrcy I merged it, but please confirm that this requires a custom postgres service and not the "native" one which comes with Easypanel.

**Comment by DrMxrcy** [comment]
If you have any more questions let me know!

**Pull request merged**